frontend/pages/admin/OrgSettingsPage/cards/Smtp/Smtp.tsx

```diff
@@ -289,28 +326,7 @@ const Smtp = ({
   return (
     <div className={baseClass}>
       <div className={`${baseClass}__section`}>
-        <SectionHeader
-          title="SMTP options"
-          details={
-            !sesConfigured ? (
-              <small
-                className={`smtp-options smtp-options--${
-                  appConfig.smtp_settings?.configured
-                    ? "configured"
-                    : "notconfigured"
-                }`}
-              >
-                <em>
-                  {appConfig.smtp_settings?.configured
```

Comment:
Can you link the product decision to remove this?

Reply:
https://github.com/fleetdm/fleet/issues/24660#issuecomment-2561362083